src/routes/concepts/components/basics.mdx

```diff
@@ -144,11 +122,10 @@ function MyComponent() {
 }
 ```
 
-This example uses a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) to conditionally render different content based on the value of `count`.
-When `count` is greater than 5, the component will display `"Count limit reached"`.
-Otherwise, it will display the current count with an increment button.
+This example uses a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) to conditionally render different content based on the value of `count`. When `count` is greater than 5, the component will display `"Count limit reached"`. Otherwise, it will display the current count with an increment button.
 
 :::info
+ 
 To simplify conditional rendering, Solid provides built-in [control-flow](/concepts/control-flow/conditional-rendering) components like [`Show`](/concepts/control-flow/conditional-rendering#show), which create a more readable conditional rendering experience.
 
     ```tsx
```

Comment:
I'm not sure why this code block is indented. The indentation probably should be removed.